config/admin-cli.properties

```diff
@@ -19,11 +19,43 @@ cli.controller.grpc.uri=tcp://localhost:9090
 cli.store.metadata.backend=segmentstore
 
 # General configuration for the Pravega cluster at hand.
-pravegaservice.cluster.name=pravega/pravega
+pravegaservice.clusterName=pravega/pravega
```

Comment:
Why was this reverted?

Reply:
This was wrong already. Name should be `clusterName` as used here (https://github.com/pravega/pravega/blob/5a405999653ccfcc9e1633261248da7725d06ce2/segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceConfig.java#L54)).